dom.go

```diff
@@ -244,7 +260,9 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 	}
 	style := h.Node.Get("style")
 	for name, value := range h.styles {
-		style.Call("setProperty", name, value)
+		for value, _ := range value {
+			style.Call("setProperty", name, value)
+		}
```

Comment:
I don't understand this change. This causes us to call `setProperty` with the same `name` but different `value` multiple times?